src/tests/ftest/nvme/enospace.py

```diff
@@ -375,18 +402,16 @@ def test_performance_storage_full(self):
                       .format(max_mib_baseline, max_mib_latest))
 
     def test_enospace_no_aggregation(self):
-        """Jira ID: DAOS-4756.
+        """ Jira ID: DAOS-4756.
```

Comment:
Actually, there shouldn't be a space
```suggestion
        """Jira ID: DAOS-4756.
```

Reply:
- [ ] Fix docstring format

Reply:
Fixed with commit 210648f04